### ISSUE TITLE
(Keyboard Nav) Hitting [Esc] does not exit the Keyboard Shortcuts Menu

### ISSUE THREAD

**Issue by stemilymill** [phetsims/gravity-force-lab-basics#299]
https://github.com/phetsims/qa/issues/702
Win10 Firefox and Chrome

Hitting [Esc] does not exit the Keyboard Shortcuts Menu.

The only exception I have found is:
Open sim
[tab] until keyboard shortcut icon on nav bar
[space] to select
Without doing anything else, [Esc] to exit menu

One example to recreate the bug:
Click keyboard shortcuts icon on nav bar
Hit [Esc]
nothing will happen

Another example:
Using keyboard nav or clicking, open keyboard shortcuts menu
hit [tab]
hit [Esc]
nothing will happen

This occurs whether or not voicing is turned on.

**EDIT**
This also happens with the Preferences menu dialog

**Comment by Nancy-Salpepi** [comment]
To add on to what @stemilymill was reporting, the esc key works to exit out of the preferences tab ONLY if you haven't manipulated anything after it is opened. Once you click on anything, the esc key no longer works.

**Comment by Nancy-Salpepi** [comment]
When testing this morning on the iMac + safari, the escape key was working as it should. I went back and tested on the MacBook + chrome (where the issue was occurring last week) and it is also working there as it should (with and without voicing on). Not sure if issue still occurs with Windows.

**Comment by stemilymill** [comment]
I just tested on WIn10 chrome and the escape key was working properly.

**Comment by jessegreenberg** [comment]
I am guessing that what is happening in this issue is that "esc" to close the dialog is only happening if focus is in the dialog. Focus is in the dialog when you first open it, or while you are tabbing through focusable dialog components. But as soon as focus leaves the dialog (by tabbing out of it, or clicking anywhere inside of it) focus goes away and you cannot close it with the "esc" key.

@Nancy-Salpepi or @stemilymill does that sound like an accurate description of what is happening?

**Comment by stemilymill** [comment]
I first noticed it when I was using nav keys to go through the keyboard shortcuts menu with voicing- I think that means the focus was still in the dialog? But it also happened after clicking inside of the dialog. However, this issue doesn't seem to be happening anymore for me or Nancy.

**Comment by jessegreenberg** [comment]
OK thanks @stemilymill - If you were using nav keys, I think that means you were using VoiceOver, not Voicing. In that case it is possible you were reading thorugh the dialog while focus was still outside of it.

But since it doesn't seem to be happening anymore I don't think there is anything else to check, closing this issue.
